draft-js-plugins-editor/src/Editor/index.js

```diff
@@ -64,195 +69,61 @@ export default class PluginEditor extends Component {
-  };
-
-  handleDroppedFiles = (selection, files) => {
-    if (this.props.handleDroppedFiles) {
-      const result = this.props.handleDroppedFiles({
-        selection,
-        files,
-        getEditorState: this.getEditorState,
-        updateEditorState: this.onChange,
-        props: this.props,
-      });
-      if (result) {
-        return result;
-      }
-    }
-
-    return this.plugins
-        .map((plugin) => {
-          if (plugin.handleDroppedFiles) {
-            const result = plugin.handleDroppedFiles({
-              selection,
-              files,
-              getEditorState: this.getEditorState,
-              updateEditorState: this.onChange,
-              props: this.props,
-            });
-            if (result) {
-              return result;
-            }
-          }
-
-          return undefined;
-        })
-        .find((result) => result !== undefined);
-  };
-
-  handleDrop = (selection, dataTransfer, isInternal) => {
-    if (this.props.handleDrop) {
-      const result = this.props.handleDrop({
-        selection,
-        dataTransfer,
-        isInternal,
-        getEditorState: this.getEditorState,
-        updateEditorState: this.onChange,
-        props: this.props,
-      });
-      if (result) {
-        return result;
-      }
-    }
-
-    return this.plugins
-        .map((plugin) => {
-          if (plugin.handleDrop) {
-            const result = plugin.handleDrop({
-              selection,
-              dataTransfer,
-              isInternal,
-              getEditorState: this.getEditorState,
-              updateEditorState: this.onChange,
-              props: this.props,
-            });
-            if (result) {
-              return result;
-            }
-          }
-
-          return undefined;
-        })
-        .find((result) => result !== undefined);
-  };
-
-  // Put the keyboard focus on the editor
-  focus = () => {
-    this.refs.editor.focus();
-  };
-
-  createHandleListener = (name) => (event) => (
-    this.plugins
-      .filter((plug) => plug[name])
-      .map((plugin) => plugin[name](event))
-      .find((result) => result === true) === true
-  );
-
-  createOnListener = (name) => (event) => (
-    this.plugins
-      .filter(plug => typeof plug[name] === 'function')
-      .forEach(plug => plug[name](event))
-  );
-
-  createEventListeners = () => {
-    const listeners = {
-      onChange: this.onChange,
-      handleKeyCommand: this.handleKeyCommand,
-      keyBindingFn: this.keyBindingFn,
-      handleReturn: this.handleReturn,
-    };
-
-    const keepHandlers = ['onChange', 'handleKeyCommand'];
-
-    // bind random onListeners and handleListeners
-    this.plugins.forEach((plug) => {
-      Object.keys(plug).forEach((attrName) => {
-        if (attrName.indexOf('on') === 0 && keepHandlers.indexOf(attrName !== -1)) {
-          listeners[attrName] = this.createOnListener(attrName);
+        if (attrName.indexOf('handle') === 0) {
```

Comment:
would be nice if draft-js would let us import the hook names rather than having to do stuff like this